families/settings/tests/test_tp_settings.py

```diff
@@ -16,6 +16,9 @@
 import hashlib
 import base64
 
+from sawtooth_processor_test.transaction_processor_test_case \
```

Comment:
Similar question as earlier, is this change a lint/formatting issue? There are few other files below this where only the order in which import statements are written is changed.